### ISSUE TITLE
Configurable export/import

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardForAndroid#1652]
Original post:
https://forum.xda-developers.com/showpost.php?p=74839356&postcount=2939

The point is that we should let the user decide what exactly should be exported or imported.

Something like that (exact items should be discussed):
```
[x] Settings
[x] Apps Mgmt rules
[x] Filters
[x] User filter
[x] Whitelist
```

**Comment by CrescendoFang** [comment]
and how to export/import settings on v3.0.xxxx?

**Comment by TPS** [comment]
@CrescendoFang Settings > "⋮" Menu on ↗️ corner⁉

**Comment by CrescendoFang** [comment]
its used to be in the general settings page... lol....

**Comment by vozersky** [comment]
merged into #3378
